lib/note-detail.jsx

```diff
@@ -3,6 +3,7 @@ import marked from 'marked';
 import Textarea from 'react-textarea-autosize';
 import { noop, get, debounce } from 'lodash';
 import analytics from './analytics'
```

Comment:
First off, thanks for the PR! This makes sense to me to add. Not your fault, but would you mind adding a semicolon to the previous line? I know it's not needed but it looks off with the new import you added.